.lgtm.yml

```diff
@@ -0,0 +1,27 @@
+path_classifiers:
+  docs:
+    - docs
+  test:
+    - "**/TESTS"
+    - "**/UNITTESTS"
+  tools:
+    - tools
+
+extraction:
+  cpp:
+    prepare:
+      packages:
+        - ninja-build
+        - python-pip
+    after_prepare:
+      - pip install --user cmake
```

Comment:
Could we install CMake from the kitware PPA instead of PyPI? Sometimes the PyPI release seems to lag behind the latest. Also means one less package to install as we wouldn't need `pip`.

Reply:
From https://lgtm.com/help/lgtm/analysis-faqs "There is no way to install packages from external repositories (PPAs) since that would allow privilege escalation."

Reply:
Fair enough.